.travis.yml

```diff
@@ -13,6 +13,11 @@ git:
 services:
   - docker
 
+addons:
+  apt:
+    packages:
+    - bc
```

Comment:
Required?

Reply:
while looking fir it, I realized it's in the Docker that it's required
I added a guard to avoid pushing too big version numbers https://github.com/opengisch/QField/blob/12512a07941b524863a53c0642819943f48ae1cf/scripts/docker-build.sh#L62

Reply:
https://github.com/opengisch/OSGeo4A/pull/57